src/Jira/Api.php

```diff
@@ -347,6 +348,33 @@ public function getVersions($projectKey)
     }
 
     /**
+     * Helper method to find a specific version based on the name of the version.
+     *
+     * @param string $projectKey Project Key
+     * @param string $name The version name to match on
+     *
+     * @return int|null VersionId on match or null when there is no match
+     */
+    public function findVersionByName($projectKey, $name)
```

Comment:
I recommend setting `private` scope for new methods, that you don't plan to expose in the API yet.

Adding all stuff public by default work create more problems down the road, when we release 2.0 version and need to maintain that API to avoid BC breaks.